ch_backup/ch_backup.py

```diff
@@ -104,6 +106,22 @@ def list(self, state: BackupState = None) -> Sequence[BackupMetadata]:
 
         return backups
 
+    def _lock_id(self, operation: str) -> Optional[str]:
+        lock_id = operation
+        # if the replica name part of the id starts with '$' it means
+        # we used `platform.node()`
+        missing_shard = "/UNKNOWN"
+        missing_replica = f"/${platform.node()}"
```

Comment:
In case that getting replica and shard name through the query fails. It would be nice to still have some identifier for the locking node. Would `platform.node()` work for this purpose?